src/Uno.UI/UI/Xaml/Controls/TimePicker/NativeTimePickerFlyout.Android.cs

```diff
@@ -16,9 +16,16 @@ partial class NativeTimePickerFlyout
 	private TimeSpan _initialTime;
 
 	internal bool IsNativeDialogOpen => _dialog?.IsShowing ?? false;
+	internal UnoTimePickerDialog GetNativeDialog() => _dialog;
+
 
 	internal protected override void Open()
 	{
+		if (Time.Ticks == -1)
```

Comment:
Final nitpick :)

We should refactor all uses of `-1` in the tests and actual code behind some sort of internal const int, similar to how DatePicker has it: https://github.com/unoplatform/uno/blob/7b287950e524d4b722c05dba37ed6fedbd226e10/src/Uno.UI/UI/Xaml/Controls/DatePicker/DatePicker.cs#L25